### PR TITLE
Fix open block add child

### PR DESCRIPTION
**Pull request by sid597**
Draft Fix for #1502

### CODE REVIEW

**Comment by neotyk** [comment]
Please fix `lint` & `style` checks.

**Comment by sid597** [comment]
@neotyk @filipesilva Fixed style and lint, ready to merge

**Comment by filipesilva** [comment]
There's no reviews on this pr atm, who do you think is the most appropriate reviewer to take a look?

**Comment by sid597** [comment]
I think Alex already reviewed this PR when he tried this locally

File: src/cljc/athens/common_events.cljc [hunk 1]
```diff
@@ -177,14 +177,20 @@
...
-     :event/args    {:parent-uid parent-uid
-                     :new-uid    new-uid
-                     :add-time?  add-time?}}))
+  ([last-tx parent-uid new-uid] (let [event-id (gen-event-id)]
+                                  {:event/id      event-id
+                                   :event/last-tx last-tx
+                                   :event/type    :datascript/add-child
+                                   :event/args    {:parent-uid parent-uid
+                                                   :new-uid    new-uid
+                                                   :add-time?  false?}}))
```
Comment (juniusfree) [inline src/cljc/athens/common_events.cljc:186]: Should this be `false` instead of `false?`

Comment (sid597) [reply src/cljc/athens/common_events.cljc:186]: Thank you Junius, yeah it should be

Comment (filipesilva) [reply src/cljc/athens/common_events.cljc:186]: Good catch, very subtle bug!

**Pull request merged**